server/views/pages/admin/admin.njk

```diff
@@ -0,0 +1,38 @@
+{% extends "layout.njk" %}
```

Comment:
At the moment this screen is only intended for testing purposes - hence the basic design and  no accompanying unit tests